src/index.html

```diff
@@ -9,6 +9,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
```suggestion
      <a href="#home" class="header__logo">
        <img src="./images/logo.png" alt="logo">
      </a>
```
format it like this for better readability